app/components/shared/sdg/goal_stats_component.html.erb

```diff
@@ -0,0 +1,43 @@
+<div class="sdg-goal-stats">
+
+  <h3><%= goal.code_and_title %></h3>
+
+  <div class="stats-numbers">
+
+    <div class="small-12 medium-4 column">
```

Comment:
Maybe we could remove some duplication :thinking:. Here's one option:

```erb
  <div class="stats-numbers">
    <% stats.each do |text, amount, options = {}| %>
      <div class="small-12 medium-4 column">
        <%= tag.p(options) do %>
          <%= text %> <br><span class="number"><%= amount %></span>
        <% end %>
      </div>
    <% end %>
```

And then, in the component:

```ruby
  def stats
    [
      [t("sdg.stats.polls"), goal.polls.count],
      # ...
      [t("sdg.stats.budget_investments.sent"), goal.budget_investments.count, class: "sent"]
      # ...
    ]
  end
```